src/App.vue

```diff
@@ -1,33 +1,37 @@
 <template>
-  <div v-if="true"><Nav/></div>
-  <div v-else><NavReg/></div>
+  <div v-if="is_authenticated()"><Nav /></div>
+  <div v-else><NavReg /></div>
 
-  <div v-if="alert.message" :class="`alert ${alert.type}`">{{alert.message}}</div>
+  <div v-if="alert.message" :class="`alert ${alert.type}`">
+    <Alert />
```

Comment:
How are you getting the message in the component ? Props from this component has not been passed onto this child component `Alert`.